app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    content = command.split()
+    command, old_file, new_file = content[0], content[1], content[2]
```

Comment:
You can do it un one line, but before you need to check that `command` length == 3

Reply:
But command is a string, length of command always > 3